src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -50,6 +51,7 @@ type Props = {
     currentUser?: User,
     defaultView: string,
     detailsSidebarProps: DetailsSidebarProps,
+    docgenPreviewSidebarProps: DocgenSidebarProps,
```

Comment:
should `Preview` be removed from this name or is this intentional? i.e. is this related to ContentPreview or some other use of the word?